tarantool_test.go

```diff
@@ -352,6 +356,52 @@ func BenchmarkClientParallelMassiveUntyped(b *testing.B) {
 	close(limit)
 }
 
+func BenchmarkConnection_Replace(b *testing.B) {
+	conn, err := Connect(server, opts)
+	if err != nil {
+		b.Errorf("No connection available")
+		return
+	}
+	defer conn.Close()
+	spaceNo = 520
+
+	b.ResetTimer()
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			_, err := conn.Replace(spaceNo, []interface{}{uint(1), "hello", []interface{}{}})
+			if err != nil {
+				b.Log(err)
+				b.Errorf("No connection available")
+			}
+		}
+	})
+}
+
+func BenchmarkConnection_LargeSelect(b *testing.B) {
+	conn, err := Connect(server, opts)
+	if err != nil {
+		b.Errorf("No connection available")
+		return
+	}
+	defer conn.Close()
+
+	schema := conn.Schema
+	rSpaceNo, rIndexNo, err := schema.ResolveSpaceIndex("test_perf", "secondary")
```

Comment:
Oh, that seems nice. Why don't use it on all of your bench tests?

Reply:
Fixed.